cli/azd/cmd/init.go

```diff
@@ -108,6 +109,7 @@ type initAction struct {
 func newInitAction(
 	accountManager account.Manager,
 	userProfileService *azcli.UserProfileService,
+	_ auth.EnsureLoginGuard,
```

Comment:
```suggestion
	auth.EnsureLoginGuard,
```

Reply:
Does this actually work? We're not consuming a dependency; we're including a side-effect constructor parameter.

Reply:
Looks like it gives: `mixed named and unnamed parameters`.  I though we could mix it for input.  neverminded then